### ISSUE TITLE
LUA Error - MinimapZoomIn issue

### ISSUE THREAD

**Issue by Dranni21312** [Ketrel/MinimapButtonFrame-Redux#7]
Here's a LUA error being thrown in the latest version of MBF-Redux (2.0.3). The latest Ace3 for 10.0.0 is installed.

```
7x .../MinimapButtonFrameRedux/MinimapButtonFrameRedux.lua:1935: attempt to index global 'MinimapZoomIn' (a nil value)
[string "@Interface/AddOns/MinimapButtonFrameRedux/MinimapButtonFrameRedux.lua"]:1935: in function `MBFC_KeepBlizzHidden'
[string "@Interface/AddOns/MinimapButtonFrameRedux/MinimapButtonFrameRedux.lua"]:1838: in function `?'
[string "@Interface/AddOns/Ace3/AceTimer-3.0-17/AceTimer-3.0.lua"]:55: in function <Interface/AddOns/Ace3/AceTimer-3.0/AceTimer-3.0.lua:50>

Locals:
(*temporary) = nil
(*temporary) = nil
(*temporary) = "attempt to index global 'MinimapZoomIn' (a nil value)"

```

Since the API has changed, I believe there are changes to MinimapZoomIn, MinimapZoomOut, MiniMapMailFrame, MiniMapBattlefieldFrame, etc. (most buttons have been moved to the Minimap namespace, e.g. Minimap.ZoomIn, Minimap.ZoomOut, etc.)

**Comment by Ketrel** [comment]
This should be fixed in v2.0.4

https://github.com/Ketrel/MinimapButtonFrame-Redux/releases/tag/v2.0.4

**Comment by Dranni21312** [comment]
Yep. That fixed it! Thank you
